dags/mapper_dag.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+import json
+import sys
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from rikolti.metadata_mapper.lambda_shepherd import get_vernacular_pages, get_collection
+from rikolti.metadata_mapper.lambda_function import map_page as lambda_map_page
+
+
+@task()
+def fetch_pages(params=None):
+    if not params:
+        return False
+
+    payload = params.get('payload')
+    collection_id = payload.get('collection_id', 0)
+    # raise an error?
+    if not collection_id:
+        return []
+
+    pages = get_vernacular_pages(
+                collection_id)
+
+    return pages
+
+@task()
+def map_page(page: str, params=None):
+    if not params:
+        return False
+
+    payload = params.get('payload')
+    collection_id = payload.get('collection_id', 0)
+    # raise an error?
+    if not collection_id:
+        return {}
+
+    collection = get_collection(collection_id)
+    payload.update({'collection': collection})
+    payload.update({'page_filename': page})
```

Comment:
yikes, yeah I think we want this to be different.

Reply:
I refactored the code so that only `collection_id` is required as input for running the mapper. Eager to hear your feedback.

Reply:
After all of the refactoring, `lambda_shepherd` and `lambda_function` are definitely misnamed! I made the assumption that we no longer need lambda entry points, so payload input isn't supported by the code anymore. (Let me know if there's some reason we should maintain this after all).

How about this renaming:
lambda_shepherd.py --> map_collection.py
lambda_function.py --> map_page.py